source code

```diff
@@ -6,13 +6,13 @@
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
     <script src="jquery-2.1.4.js"></script>
     <link rel="stylesheet" type="text/css" href="styles.css" />
+    <script type="text/javascript" src="scripts.js"></script>
   </head>
-<body>
+  <body>
```

Comment:
We probably don't need an extra line between the closing head tag and opening body tag.

So, let's remove the extra line.

Reply:
Hey @yangc95 Please review this also. I have removed extra space.